pages/common/lein.md

```diff
@@ -0,0 +1,23 @@
+# lein
+
+> Manage clojure projects with declarative configuration.
+
+- Generate scaffolding for a new project based on a template:
+
+`lein new {{template}} {{project_name}}`
```

Comment:
Is this the path to the template or the template name? Perhaps we could change this to `{{template_name}}` or `{{path/to/template}}` as appropriate.